src/plugins/realsense/realsense_thread.cpp

```diff
@@ -50,6 +50,8 @@ RealsenseThread::init()
 	frame_id_                    = config->get_string(cfg_prefix + "frame_id");
 	pcl_id_                      = config->get_string(cfg_prefix + "pcl_id");
 	laser_power_                 = config->get_int(cfg_prefix + "device_options/laser_power");
+	restart_after_num_errors_ =
+	  config->get_uint_or_default(std::string(cfg_prefix + "restart_after_num_errors").c_str(), 50);
```

Comment:
The setting and its default should be documented in the config, otherwise it'll simply be forgotten...

Reply:
Fixed!